### PR TITLE
Eth 623

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Updates

### CODE REVIEW

File: EndaMaryamDengelat/EDMG008.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <msDesc xml:id="ms">
                     <msIdentifier>
                         <repository ref="INS1017EndaMaryamDengelat"/>
-                        <idno>EDMG-008</idno>
+                        <idno facs="EDMG/008/ EDMG-008" n="191">EDMG-008</idno>
```
Comment (eu-genia) [inline EndaMaryamDengelat/EDMG008.xml:28]: never blanks in IDs, links etc etc etc.

File: EndaMaryamDengelat/EDMG027.xml [hunk 2]
```diff
@@ -25,7 +25,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <msDesc xml:id="ms">
                     <msIdentifier>
                         <repository ref="INS1017EndaMaryamDengelat"/>
-                        <idno>EDMG-027</idno>
+                        <idno facs="EDMG/027/ EDMG-027" n="80">EDMG-027</idno>
```
Comment (eu-genia) [inline EndaMaryamDengelat/EDMG027.xml:28]: same

File: ParisBNF/et/BNFet623.xml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                    </physDesc>
+                    <history>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:bm:Strelcyn1954Griaule"/>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet623.xml:57]: bm:

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/et/BNFet623.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet623.xml]: done

File: EndaMaryamDengelat/EDMG027.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply EndaMaryamDengelat/EDMG027.xml]: done

File: EndaMaryamDengelat/EDMG008.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply EndaMaryamDengelat/EDMG008.xml]: sorry, done.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**